src/vm/codeman.cpp

```diff
@@ -1660,25 +1660,8 @@ BOOL EEJitManager::LoadJIT()
     if ((!IsCompilationProcess() || !fUseRyuJit) &&     // Use RyuJIT for all NGEN, unless we're falling back to JIT64 for everything.
         (newJitCompiler != nullptr))    // the main JIT must successfully load before we try loading the fallback JIT
     {
-        BOOL fUsingCompatJit = FALSE;
-
         if (!fUseRyuJit)
```

Comment:
Actually, the entire `#ifdef` starting with:
```
#if defined(_TARGET_X86_)
    // If COMPlus_UseLegacyJit=1, then we fall back to compatjit.dll.
    //
    // This fallback mechanism was introduced for Visual Studio "14" Preview, when JIT64 (the legacy JIT) was replaced with
```
should be deleted, since there is no remaining fallback.

---
In reply to: [113786338](https://github.com/dotnet/coreclr/pull/11262#discussion_r113786338) [](ancestors = 113786338)